script/generate-e2e-tests.js

```diff
@@ -800,6 +800,25 @@ const components = new Map([
       ],
     },
   ],
+  [
+    'NavList',
```

Comment:
@TylerJDev heads up, this might merge conflict with your open PR (depending on which one gets merged first)